src/libraries/System.Runtime.Serialization.Formatters/src/System.Runtime.Serialization.Formatters.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppCurrent)-Browser</TargetFrameworks>
     <Nullable>enable</Nullable>
-    <!-- When we replace implementations with PNSE, need to suppress some "field is never assigned to" warnings. -->
+    <!-- When we replace implementations with PNSE, need to suppress some "field is never assigned to" warnings. A -->
```

Comment:
assume this doesnt need to be included (same for `eng/illink.targets`)?

Reply:
Yes, I will revert it before merging. It is just a convenient way to trigger all tests. The alternative is trigger all tests manually in AzDo that is a lot of mouse clicks.

Reply:
ah ok, good tip.